libraries/kitsu/src/main/java/com/chesire/nekome/kitsu/ResponseParsing.kt

```diff
@@ -1,6 +1,7 @@
 package com.chesire.nekome.kitsu
 
 import com.chesire.nekome.core.Resource
+import com.chesire.nekome.datasource.auth.AuthException
```

Comment:
Reminder to self:
Kitsu depends on AuthDataSource
KitsuAuth depends on AuthDataSource